docs/templates/openj9/sso74-openj9-https.adoc

```diff
@@ -9,7 +9,7 @@
 :toc-placement!:
 :toclevels: 5
 
-An example application based on RH-SSO 7.4 on OpenJ9 image. For more information about using this template, see https://github.com/jboss-container-images/redhat-sso-openj9-7-openshift-image/docs.
+An example application based on RH-SSO 7.4 on OpenJ9 image. For more information about using this template, see https://github.com/jboss-container-images/redhat-sso-7-openshift-image/docs.
```

Comment:
Above URL returns HTTP 404 (not found). Maybe replace with `https://github.com/jboss-container-images/redhat-sso-7-openshift-image/tree/sso74-dev/docs` instead?